tests/test_unittest/test_gtc/test_gtcpp_compilation.py

```diff
@@ -51,7 +45,7 @@ def build_gridtools_test(tmp_path: Path, code: str):
     ext_module = setuptools.Extension(
         "test",
         [str(tmp_src.absolute())],
-        include_dirs=[config.GT_INCLUDE_PATH, config.build_settings["boost_include_path"]],
+        include_dirs=[gridtools.get_include_dir(), config.build_settings["boost_include_path"]],
```

Comment:
Oh right, `config` is also needed here, so we can't totally eliminate this dependency. 👍